modules/apps/content-dashboard/content-dashboard-web/src/test/java/com/liferay/content/dashboard/web/internal/item/type/DDMStructureContentDashboardItemTypeTest.java

```diff
@@ -0,0 +1,184 @@
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.json.JSONFactoryUtil;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.model.Group;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.util.LocaleUtil;
+
+import java.util.Date;
+import java.util.Locale;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import org.mockito.Mockito;
+
+/**
+ * @author Cristina González
+ */
+public class DDMStructureContentDashboardItemTypeTest {
+
+	@BeforeClass
+	public static void setUpClass() {
+		JSONFactoryUtil jsonFactoryUtil = new JSONFactoryUtil();
+
+		jsonFactoryUtil.setJSONFactory(new JSONFactoryImpl());
+	}
+
+	@Test
+	public void testCreation() throws PortalException {
+		DDMStructure ddmStructure = _getDDMStructure("structureName");
+
+		DDMStructureContentDashboardItemType
+			ddmStructureContentDashboardItemType =
+				new DDMStructureContentDashboardItemType(
+					ddmStructure, _getGroup("groupName"));
+
+		Assert.assertEquals(
+			DDMStructure.class.getName(),
+			ddmStructureContentDashboardItemType.getClassName());
+
+		Assert.assertEquals(
+			ddmStructure.getStructureId(),
+			ddmStructureContentDashboardItemType.getClassPK());
+
+		Assert.assertEquals(
+			"structureName (groupName)",
+			ddmStructureContentDashboardItemType.getFullLabel(LocaleUtil.US));
+
+		Assert.assertEquals(
+			"structureName",
+			ddmStructureContentDashboardItemType.getLabel(LocaleUtil.US));
+
+		Assert.assertEquals(
+			ddmStructure.getModifiedDate(),
+			ddmStructureContentDashboardItemType.getModifiedDate());
+
+		Assert.assertEquals(
+			ddmStructure.getUserId(),
+			ddmStructureContentDashboardItemType.getUserId());
+	}
+
+	@Test
+	public void testEquals() throws PortalException {
+		DDMStructure ddmStructure = _getDDMStructure("structureName");
+
+		DDMStructureContentDashboardItemType
+			ddmStructureContentDashboardItemType1 =
+				new DDMStructureContentDashboardItemType(
+					ddmStructure, _getGroup("groupName"));
+
+		DDMStructureContentDashboardItemType
+			ddmStructureContentDashboardItemType2 =
+				new DDMStructureContentDashboardItemType(
+					ddmStructure, _getGroup("groupName"));
+
+		Assert.assertTrue(
+			ddmStructureContentDashboardItemType1.equals(
+				ddmStructureContentDashboardItemType2));
+	}
+
+	@Test
+	public void testNoEquals() throws PortalException {
+		DDMStructure ddmStructure1 = _getDDMStructure("structureName");
+
+		DDMStructureContentDashboardItemType
+			ddmStructureContentDashboardItemType1 =
+				new DDMStructureContentDashboardItemType(
+					ddmStructure1, _getGroup("groupName"));
+
+		DDMStructure ddmStructure2 = _getDDMStructure("structureName");
+
+		DDMStructureContentDashboardItemType
+			ddmStructureContentDashboardItemType2 =
+				new DDMStructureContentDashboardItemType(
+					ddmStructure2, _getGroup("groupName"));
+
+		Assert.assertFalse(
```

Comment:
The Assert.assertNotEquals  is read as expected is not equals to actual. Since we don't have expected or actual value, I think this is no the right method to be called, since we are not asserting this two values are equal, we are checkering if the equals method works as expected.